### PR TITLE
added spaces-as-L2 feature

### PR DESCRIPTION
**Pull request by pmatulis**
Destined for master only.

### CODE REVIEW

File: en/intro-concepts.md [hunk 1]
```diff
@@ -120,11 +120,12 @@ communication. A fabric is a logical grouping of unique VLANs. A default fabric
...
-A *space* is a logical grouping of subnets that are able to communicate
-with each other. Subnets within each space need not belong to the same fabric.
-A default space ('space-0') is created when MAAS is installed and includes all
-detected subnets.
+A *space* is a logical grouping of VLANs whose subnets are able to communicate
+with one another. VLANs within each space need not belong to the same fabric.
+A default space is not created when MAAS is installed.
 
+A space's main purpose is to facilitate machine acquisition for
+[Juju](https://jujucharms.com/docs/stable/about-juju).
```
Comment (degville) [inline en/intro-concepts.md]: Just a small suggestion - do you think it's worth explaining (briefly) _how_ spaces facilitate Juju?

Comment (pmatulis) [inline en/intro-concepts.md]: If I do that I'd rather link to Juju documentation. I see [this](https://jujucharms.com/docs/2.0/reference-constraints). Do we have something better?

Comment (degville) [inline en/intro-concepts.md]: Good idea - how about this: https://jujucharms.com/docs/2.0/network-spaces

Comment (pmatulis) [inline en/intro-concepts.md]: Ok thanks. I'll use that.

**Comment by pmatulis** [comment]
Ok to merge?

**Pull request merged**